mlir-sycl/include/mlir/Dialect/SYCL/IR/SYCLOps.td

```diff
@@ -668,6 +668,27 @@ def SYCLCallOp : SYCL_Op<"call", [CallOpInterface]> {
   }];
 }
 
+////////////////////////////////////////////////////////////////////////////////
+// accessor.get_pointer OPERATION
+////////////////////////////////////////////////////////////////////////////////
+
+def SYCLAccessorGetPointerOp
```

Comment:
Can we add a verifier checking that the element type of the returned `memref` and the element type of the `accessor` are the same?

Reply:
Also, valid tests in `mlir-sycl/test/Dialect/IR/SYCL/ops.mlir` and invalid in `mlir-sycl/test/Dialect/IR/SYCL/invalid.mlir`